src/label_studio_sdk/converter/imports/coco.py

```diff
@@ -216,15 +215,17 @@ def convert_coco_to_ls(
             task[out_type][0]["result"].append(item)
 
         if "segmentation" in annotation and len(annotation["segmentation"]):
-            item = create_segmentation(
-                annotation,
-                categories,
-                segmentation_from_name,
-                image_height,
-                image_width,
-                to_name,
-            )
-            task[out_type][0]["result"].append(item)
+            for single_segmentation in annotation["segmentation"]:
```

Comment:
to avoid conflicts with other `segmentation` variables, named it as `single_segmentation`